sql-plugin/src/main/scala/com/nvidia/spark/rapids/Plugin.scala

```diff
@@ -128,12 +130,10 @@ class RapidsExecutorPlugin extends ExecutorPlugin with Logging {
         ShimLoader.setSparkShimProviderClass(conf.shimsProviderOverride.get)
       }
 
-      // we rely on the Rapids Plugin being run with 1 GPU per executor so we can initialize
-      // on executor startup.
-      if (!GpuDeviceManager.rmmTaskInitEnabled) {
```

Comment:
Why was this block removed?  It should be placed after the cudf version check.

Reply:
Sorry, Inadvertently removed it during refactoring.